index.md

```diff
@@ -361,16 +361,17 @@ and because it does a user-level install, it will not interfere with other Pytho
 If you have the choice of Python 2 or 3, choose Python 3.
 
 There are two ways to get the libraries you need:
-* You can install them in an existing Conda environment.
-* You can create a new Conda environment.
+* You can install them in an existing Conda environment (not recommended).
```

Comment:
Can we flip the order here to have the recommended option first?

Reply:
Yes, but then we might want to reverse the order of the two subsections that follow.

Reply:
Agreed - let's flip the order and add a `(not recommended)` to `#### Install libraries in an existing Conda environment`

Reply:
Ok.  Please merge this PR and I will open another one with these changes (just to keep the work flow simple).

(Or feel free to make the changes yourself.)